### ISSUE TITLE
Support suite-level test configuration via Feature tags

### ISSUE THREAD

**Issue by mallison** [badeball/cypress-cucumber-preprocessor#1158]
### Current behavior

Suite-level configuration, eg `testIsolation`, cannot be applied using tags.

Eg.

```gherkin
@testIsolation(false)
Feature: Apply Feature tags at the suite level

    Scenario: Open a page
        Given I load a page

    Scenario: the page still exists
        Then the page should exist
```

Does not work.

### Desired behavior

Tags set on a `Feature` get applied to the corresponding `describe` so that suite-level configuration can be altered.

### Test code to reproduce

I've created a small repository with a single feature file to demonstrate the issue https://github.com/marvelapp/suite-level-tags

In this repo when I run the test suite I see

`No commands were issued in this test`

for each scenario.

<img width="1840" alt="Screenshot 2024-02-19 at 12 16 00" src="https://github.com/badeball/cypress-cucumber-preprocessor/assets/265952/cdae50cb-4c92-43d7-bba6-4a8f90ffebb0">

I see from the code that this will never work because configuration set via tags is always set on the _test_, ie `it`.

If this feature request is reasonable I'm happy to create a pull request for it.

### Versions

* **Cypress version**: 13.6.4
* **Preprocessor version**: 20.0.1
* **Node version**: 13.6.4

### Checklist

- [x] I've read the [FAQ](https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/faq.md).
- [x] I've read [instructions for logging issues](https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/CONTRIBUTING.md#bug-reports).
- [x] I'm not using `cypress-cucumber-preprocessor@4.3.1` (package name has changed and it is no longer the most recent version, see [#689](https://github.com/badeball/cypress-cucumber-preprocessor/issues/689)).

**Comment by badeball** [comment]
> If this feature request is reasonable I'm happy to create a pull request for it.

That'd be fine. I recommend that you specify the desired behavior using tests before implementing anything though, and use that to propose new behavior.

**Comment by mallison** [comment]
> I recommend that you specify the desired behavior using tests

I've [create a pull request](https://github.com/badeball/cypress-cucumber-preprocessor/pull/1160) with a test showing the desired behaviour. I'd played around with implementation, as it looked like a small change, so I pushed those changes too.

**Comment by badeball** [comment]
This has been fixed with v20.0.2.
